.travis.yml

```diff
@@ -3,9 +3,6 @@ dist: trusty
 language: ruby
 cache: bundler
 before_install:
-  - if [ $BUNDLER_VERSION ]; then
```

Comment:
Is this important to preserve? I don't remember why it was added.

Reply:
The pdk sync got rid of it. Seemed reasonable and is passing an adhoc build https://jenkins-master-prod-1.delivery.puppetlabs.net/view/All/job/experimental_auto_puppetlabs-package_init-manual-parameters_adhoc/3/